app/src/routes/project/LayersPanel/index.tsx

```diff
@@ -74,17 +78,26 @@ const LayersPanel = observer(() => {
     return (
         <div
             className={clsx(
-                'border min-w-60 max-w-96 bg-black/80 backdrop-blur rounded-tr-xl shadow',
+                'fixed left-0 z-50 top-20 transition-width duration-300 opacity-100 bg-black/80',
```

Comment:
One last thing and this would be good. The transition duration is messing with the ability to drag the panel size.